openvdb/openvdb/tree/ValueAccessor.h

```diff
@@ -604,12 +623,18 @@ class CacheItem
     void addLeaf(LeafNodeType* leaf)
     {
         static_assert(!TreeCacheT::IsConstTree, "can't add a node to a const tree");
-        if (NodeType::LEVEL == 0) return;
+        if (NodeType::LEVEL == 0) {
+            mNext.addLeaf(leaf);
+            return;
+        }
```

Comment:
This was a bug - calling `addLeaf` here wouldn't do anything and the leaf ptr would leak